src/routes/v2/authenticatedSites/mediaFiles.js

```diff
@@ -43,6 +44,35 @@ class MediaFilesRouter {
     return next()
   }
 
+  async createMediaFiles(req, res, next) {
+    const { userWithSiteSessionData } = res.locals
+
+    const { directoryName } = req.params
+    const { error } = CreateMediaFilesRequestSchema.validate(req.body)
+    if (error) throw new BadRequestError(error.message)
+
+    const { files } = req.body
+    const resolvedFiles = await Promise.allSettled(
```

Comment:
I think this might not work for GitHub, because they might throw an error about some hash mismatch if the requests are sent too quickly.

For local Git, we commit (and push) on every create, which means we might encounter the Git lock when multiple requests are committing at the same time.

We might need a new method that either has the ability to retry for x number of times, or to sequentially make these create requests.

Reply:
Dang ok it's not a big deal to do this sequentially; this actually worked locally but it did get a 429